src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check `if (num === undefined)` is used to determine if the function is called without arguments. This works, but it might be more explicit to check `if (arguments.length === 0)` to ensure clarity and handle cases where `undefined` might be passed explicitly.